blox/page.cue

```diff
@@ -0,0 +1,7 @@
+{
+	title:        string
+	draft?:       bool
```

Comment:
Maybe `draft: bool | *false` instead of optional?

Reply:
yeah good